src/others/components/map/Map.tsx

```diff
@@ -39,7 +39,7 @@ export const Map = ({sourceWithLayer}: MapProps) => {
       setPopupInfo({
         longitude: event.lngLat.lng,
         latitude: event.lngLat.lat,
-        description: requestData ? `${requestData.category}: ${requestData.amount}` : 'Information unavailable'
+        description: requestData ? requestData.description : 'Information unavailable',
```

Comment:
Kinda nit but you can do that instead

```suggestion
        description: requestData?.description ?? 'Information unavailable',
```